app/dictionary.py

```diff
@@ -0,0 +1,83 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50,
```

Comment:
There is an extra comma after the 'power' key in the 'weapon' dictionary for 'lancelot'. This is not a syntax error in Python, but it's good practice to remove it for consistency.